sdks/python/apache_beam/examples/avro_bitcoin.py

```diff
@@ -38,7 +38,9 @@
 
 # pylint: disable=wrong-import-order, wrong-import-position
 try:
-  from avro.schema import Parse # avro-python3 library for python3
+  from avro.schema import Parse as avro_parser # avro-python3 library for python3
+  Parse = lambda x: avro_parser(x.decode("utf-8")) \
```

Comment:
Is this good practice?

Reply:
It's recommended not to design APIs that accept both text and binary data, see: https://docs.python.org/3/howto/pyporting.html. 
Do we understand why the type of input is inconsistent? If textual data is passed in as `bytes`, can we decode them earlier, before calling `Parse`?

Reply:
Yes, I agree, thanks. I will adjust this after I make the problem described in my comment (above) work, otherwise the error won't be completely resolved :-). Would it be possible to also make a recommendation on that? Thanks – Enjoy your weekend

Reply:
We tried to address comparing types in https://github.com/apache/beam/pull/6602, but had to roll it back due to https://issues.apache.org/jira/browse/BEAM-5744. Could you take a look at https://issues.apache.org/jira/browse/BEAM-5744? If we fix that, I think we could move forward with https://github.com/apache/beam/pull/6602. See also: https://issues.apache.org/jira/browse/BEAM-5621?focusedCommentId=16649857&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16649857